jorgeRafaelSJ/passion_project.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset = "UTF-8">
+		<style type = "text/css">
+			body {
+				max-width: 1000px;
+				margin: 20px auto 20px auto;
+			}
+			h1, h2 {
+				border-bottom: 1px solid black;
+			}
+
+		</style>
+	</head>
+	<body>
+		<h1>Passion Project</h1>
+
+		<p>I have been thinking for some time about figuring out a way to make classical music more accessible for curious new listeners. Most people are turned off by the "pretentiousness" of the art environment because of how obscure it can seem. I would love to come up with a solution that can show a curious listener that:</p>
```

Comment:
Hi Travis! Just seeing your comments now. Yes, I am into classical, a lot. I did my undergrad in Piano Performance. I would love to work on something music related. :)